class/class4.1/src/PizzaBuilder.cpp

```diff
@@ -0,0 +1,10 @@
+#include "PizzaBuilder.hpp"
+
+PizzaBuilder& PizzaBuilder::addIngredient(Ingredient&& ingredient) {
+    this->pizza.addIngredient(std::move(ingredient));
+    return *this;
+}
+
+Pizza PizzaBuilder::build() const {
+    return this->pizza;
```

Comment:
you should reset internal pizza object after build call to allow creation of the next pizza. you can use std::move or local variable and std::swap